apps/prairielearn/src/tests/instructorAssessment.test.ts

```diff
@@ -528,10 +530,9 @@ describe('Instructor assessment editing', function () {
       locals.$ = cheerio.load(page);
     });
     it('should have CSRF token for testing', function () {
-      elemList = locals.$('input[name="__csrf_token"]');
+      elemList = locals.$('#test_csrf_token');
```

Comment:
The EJS created an input to be used only for testing. The same information, however, is already provided in dev mode (used in testing) to provide the same info, so that info was unnecessary.